_docs/cluster/manual/operating-system-requirements.md

```diff
@@ -54,6 +54,8 @@ sed -i --follow-symlinks 's/#PermitRootLogin yes/PermitRootLogin no/g' /etc/ssh/
 
 ## Use iptables for Bridged Network Traffic
 
+{% include alert/note.html content="This step is only necessary for EL7 and EL8 hosts." %}
```

Comment:
do we know this for sure?

Reply:
Did our ATLAS colleague have to do anything with `iptables` on their Rocky9 boxes?